pages/cockpit/user-guide/manage-environments.adoc

```diff
@@ -38,6 +37,11 @@ image:cockpit/environment-settings.png[]
 . Click *SAVE*.
 . Update the *Human Readable ID* and click *CHANGE HRID*.
 
+== Delete the environment
+
+. Go to the environment settings page.
+. Click *DELETE THIS ENVIRONMENT*. 
```

Comment:
Maybe we should add a note to explain that an environment can not be deleted if it has linked installations